contrib/docker/solr/solrconfig.xml

```diff
@@ -0,0 +1,1462 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Yes, about 80% is comments, default solrconfig for solr 6.0 don't use the schema.xml, it uses a managed-schema
This modification enables the schema.xml